app/controllers/admin/profiles/admin_profiles_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Admin
+  module Profiles
+    class AdminProfilesController < Admin::BaseController
+      skip_after_action :verify_policy_scoped, only: %i[show destroy]
+      skip_after_action :verify_authorized
+
+      def index
+        authorize AdminProfile, :index?
+        admin_profiles = policy_scope(AdminProfile)
+        @admin_profiles = Kaminari.paginate_array(admin_profiles).page(params[:page]).per(20)
```

Comment:
I really don't think the list of admins will get large enough to justify paginating.

Reply:
Done